### ISSUE TITLE
pandora.py structure

### ISSUE THREAD

**Issue by JasonLG1979** [TingPing/pandora-rest#2]
In our current implementation Pandora is passed into each object so methods can be called from the object. Do you want to keep doing that? The alternative is ofc to not and just passing the object to the method from outside the object.  Basically ```self.current_song.ban() vs self.session.ban_song(self.current_song)``` I kinda like the 2nd option, it assumes less and is coupled looser. I would rather the Track and Station objects be more property dumpsters then objects with function/methods (outside of what comes with GObject ofc).

**Comment by TingPing** [comment]
I was thinking about that but I don't really know which is better. I think i probably actually prefer `Track.ban()`. The only reason I think that would be worth avoiding is if we had multple `Client`'s floating around and the same `Track` could be used on both but I don't expect us to ever have that usage?

**Comment by JasonLG1979** [comment]
> I don't expect us to ever have that usage?

No, I really can't think of a reason why you would have more then one session at a time? Wouldn't be much of a point, who's going to want to listen to 2 songs at once?

**Comment by JasonLG1979** [comment]
Keeping the methods out of the objects just makes it seem cleaner to me, like I said it doesn't assume relationship. To me deciding how things fit together is the job of the next layer up. But that's just me.

**Comment by TingPing** [comment]
I'm ok with everything being in `Client`, if we ever want to change that it would be easy anyway.
